README.md

```diff
@@ -35,5 +35,21 @@ Or in your script:
           
 ## Example
 
-        gluegun generate https://github.com/gluegun/site_sample_config.yml
+        gluegun generate https://github.com/gluegun/site.yml
         
+## Building Gluegun Gem
+Build the gem using the command 
+
+```
+gem build gluegun.gempspec
```

Comment:
Small typo here, it should be `gluegun.gemspec` instead of `gluegun.gempspec` (we just need to remove the letter p between gem and spec)